src/squidpy/experimental/im/_detect_tissue.py

```diff
@@ -17,9 +17,9 @@
 from spatialdata.models import Labels2DModel
 from spatialdata.transformations import get_transformation
 
-from squidpy._utils import _get_scale_factors, _yx_from_shape
+from squidpy._utils import _ensure_dim_order, _get_scale_factors, _yx_from_shape
 
-from ._utils import _flatten_channels, _get_image_data
+from ._utils import _flatten_channels, _get_element_data
 
 
 class DETECT_TISSUE_METHOD(enum.Enum):
```

Comment:
I just noticed this Enum. I think the Enum classes themselves are supposed to be in UpperCamelCase.

https://docs.python.org/3/howto/enum.html
> Because Enums are used to represent constants, and to help avoid issues with name clashes between mixin-class methods/attributes and enum names, we strongly recommend using UPPER_CASE names for members, and will be using that style in our examples.